STUDIO/org.openl.rules.webstudio/src/org/openl/rules/rest/UsersController.java

```diff
@@ -254,19 +259,20 @@ public UserProfileModel getUserProfile() {
         User user = userManagementService.getUser(username);
 
         return new UserProfileModel().setFirstName(user.getFirstName())
-            .setLastName(user.getLastName())
-            .setEmail(user.getEmail())
-            .setShowHeader(IXlsTableNames.VIEW_DEVELOPER
-                .equals(userSettingsManager.getStringProperty(user.getUsername(), TABLE_VIEW)))
-            .setShowFormulas(userSettingsManager.getBooleanProperty(user.getUsername(), TABLE_FORMULAS_SHOW))
-            .setTestsPerPage(userSettingsManager.getIntegerProperty(user.getUsername(), TEST_TESTS_PERPAGE))
-            .setTestsFailuresOnly(userSettingsManager.getBooleanProperty(user.getUsername(), TEST_FAILURES_ONLY))
-            .setTestsFailuresPerTest(userSettingsManager.getIntegerProperty(user.getUsername(), TEST_FAILURES_PERTEST))
-            .setShowComplexResult(userSettingsManager.getBooleanProperty(user.getUsername(), TEST_RESULT_COMPLEX_SHOW))
-            .setShowRealNumbers(userSettingsManager.getBooleanProperty(user.getUsername(), TRACE_REALNUMBERS_SHOW))
-            .setDisplayName(user.getDisplayName())
-            .setUsername(user.getUsername())
-            .setExternalFlags(user.getExternalFlags());
+                .setLastName(user.getLastName())
+                .setEmail(user.getEmail())
+                .setShowHeader(IXlsTableNames.VIEW_DEVELOPER
+                        .equals(userSettingsManager.getStringProperty(user.getUsername(), TABLE_VIEW)))
+                .setShowFormulas(userSettingsManager.getBooleanProperty(user.getUsername(), TABLE_FORMULAS_SHOW))
+                .setTestsPerPage(userSettingsManager.getIntegerProperty(user.getUsername(), TEST_TESTS_PERPAGE))
+                .setTestsFailuresOnly(userSettingsManager.getBooleanProperty(user.getUsername(), TEST_FAILURES_ONLY))
+                .setTestsFailuresPerTest(userSettingsManager.getIntegerProperty(user.getUsername(), TEST_FAILURES_PERTEST))
+                .setShowComplexResult(userSettingsManager.getBooleanProperty(user.getUsername(), TEST_RESULT_COMPLEX_SHOW))
+                .setShowRealNumbers(userSettingsManager.getBooleanProperty(user.getUsername(), TRACE_REALNUMBERS_SHOW))
+                .setTreeView(userSettingsManager.getStringProperty(user.getUsername(), RULES_TREE_VIEW))
+                .setDisplayName(user.getDisplayName())
+                .setUsername(user.getUsername())
+                .setExternalFlags(user.getExternalFlags());
```

Comment:
Configure `Eclipse Code Formatter` plugin with `OpenL_Code_Style.xml` and reformat code with it